trs_filer/ga4gh/trs/server.py

```diff
@@ -41,9 +41,27 @@ def toolsIdGet(
 @log_traffic
 def toolsIdVersionsGet(
     id: str
-) -> List:
-    """ List versions of a tool. """
-    return []  # pragma: no cover
+) -> List[Dict]:
+    """List versions of a tool.
+    Args:
+        id: A unique identifier of the tool.
+    Returns:
+        List of version dicts corresponding given tool id.
+    Raise:
+        NotFound if no object mapping with given id present.
+    """
+
```

Comment:
I think you can remove all these empty lines. For nice small functions/methods (as they should be!!) :+1: , I don't think it helps readability - rather the opposite. But no need to do this in this PR, you can do it for all methods in the next.